napari/utils/transforms/transforms.py

```diff
@@ -135,11 +157,15 @@ def inverse(self) -> 'TransformChain':
         return TransformChain([tf.inverse for tf in self[::-1]])
 
     @property
-    def _is_diagonal(self):
+    def _is_diagonal(self) -> bool:
         if all(getattr(tf, '_is_diagonal', False) for tf in self):
             return True
         return getattr(self.simplified, '_is_diagonal', False)
 
+    @property
+    def _is_permutation(self) -> bool:
+        return getattr(self.simplified, '_is_permutation', False)
```

Comment:
```suggestion
        return self.simplified._is_permutation
```

As `_is_permutation` is defined on the base class, can we simplify?